docs/understanding-airbyte/supported-data-types.md

```diff
@@ -0,0 +1,166 @@
+| Integer                                                        | `{"type": "integer"}`                                                                    | `42`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Big integer (unrepresentable as a 64-bit two's complement int) | `{"type": "string", "airbyte_type": "big_integer"}`                                      | `"123141241234124123141241234124123141241234124123141241234124123141241234124"`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| Number                                                         | `{"type": "number"}`                                                                     | `1234.56`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| Big number (unrepresentable as a 64-bit IEEE 754 float)        | `{"type": "string", "airbyte_type": "big_number"}`                                       | `"1,000,000,...,000.1234"` with 500 0's, i.e. `"100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.1234"` |
```

Comment:
Nitpick. The actual number can be omitted I think. It already says that this is a number with 500 `0`s. Writing out the full number makes the table layout look bad.